### PR TITLE
Wire translation update.

### PR DESCRIPTION
**Pull request by simoneduca**
Not working yet. I get this (I left the log for `changes`)
screen shot 2017-07-07 at 12 35 28

### CODE REVIEW

File: src/containers/ProjectContentsContainer.jsx [hunk 1]
```diff
@@ -50,16 +52,26 @@ class ProjectContentsContainer extends Component {
...
 
+  handleChange(event) {
+    this.setState({
+      translationText: event.target.value,
+    });
+  }
+
   handleClick(event) {
     this.setState({
+      field: event.target.textContent.split(':')[0],
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: `event.target.getAttribute('data-translation-key')` would be better here. Then add that attribute to editable fields in the view eg. `data-translation-key='description'`. You started on that in #4.

File: src/containers/ProjectContentsContainer.jsx [hunk 2]
```diff
@@ -50,16 +52,26 @@ class ProjectContentsContainer extends Component {
...
+      fieldText: event.target.textContent.split(':')[1],
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: You can look up the text from the original resource in state, once you know the field name.

File: src/containers/ProjectContentsContainer.jsx [hunk 3]
```diff
@@ -50,16 +52,26 @@ class ProjectContentsContainer extends Component {
...
     });
   }
 
   handleSubmit(event) {
     event.preventDefault();
-    console.log('submit');
+    const { field, translationText } = this.state;
+    const { actions, resource } = this.props;
+    const translation = resource.translation;
+    actions.updateTranslation(translation, field, translationText);
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: This looks good to me 👍

**Comment by simoneduca** [comment]
Updating now works for`project_contents`. Looking at workflows, etc. next.

**Comment by eatyourgreens** [comment]
:+1: I think workflow strings will work if you write the keys as `strings.T0.label` etc. but haven't actually tested this.

**Pull request merged**